meson.yaml

```diff
@@ -7,7 +7,7 @@ package:
     - license: Apache-2.0
   dependencies:
     runtime:
-      - python3
+      - python-3.12-base
```

Comment:
i think the shebang will find this dep, so you should not need to add if you have python-3.12-base added below.